packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -226,7 +221,7 @@ export class Summarizer extends EventEmitter implements ISummarizer {
     private async start(
         onBehalfOf: string,
         runCoordinator: ICancellableSummarizerController,
-        options?: Readonly<Partial<ISummarizerOptions>>): Promise<RunningSummarizer> {
+        newConfig?: ISummaryConfiguration): Promise<RunningSummarizer> {
```

Comment:
I'm not sure why overwrite (in the form of newConfig) is even needed here. We are getting full settings in constructor - they should be exactly same settings we would get here. If so, I believe we should remove it.
I think the reason it was different in the past is that we were passing only "core" settings in ctor - ISummaryConfiguration, which dd not include ISummarizerOptions. Now we properly merged all settings into one structure, and the need to pass data piecemeal is gone.

Reply:
The summarizer on demand creates a new setting with disable heuristics. 


My understanding is that someone can have an enabled  config and then call the summarizerOnDemand,  or is it not true ?

Reply:
I will follow-up in a different issue - for now I'm removing the argument but we might still have a problem if I have my config enabled, the summarizer is not running and we try to run a summarizer on demand. I will clarify on the issue.